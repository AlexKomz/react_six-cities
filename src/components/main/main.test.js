import React from "react";
import renderer from "react-test-renderer";

import Main from "./main.jsx";

const children = <div></div>;

it(`Should Main render correctly`, () => {
  const tree = renderer
    .create(
        <Main mainClasses={`page__main`}>
          {children}
        </Main>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
