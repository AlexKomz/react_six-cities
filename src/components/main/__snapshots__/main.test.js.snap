// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Main render correctly 1`] = `
<main
  className="page__main"
>
  <div />
</main>
`;
